server/etcdserver/api/v3rpc/util.go

```diff
@@ -141,6 +141,10 @@ func isRPCSupportedForLearner(req interface{}) bool {
 		return true
 	case *pb.RangeRequest:
 		return r.Serializable
+	case *pb.AuthenticateRequest:
+		return true
+	case *pb.AlarmRequest:
```

Comment:
Do we need to add AlarmRequest now

Reply:
I think we need to add this. because when we do learner endpoint health, cli will do AlarmList operation(`resp, err := cli.AlarmList(ctx)` in `etcdctl/ctlv3/command/ep_command.go`),so add `case *pb.AlarmRequest:` for fit this.

Reply:
ah got it, then this change is valid, thanks!